### DISCUSSION TITLE
How to add documentsFileUpload field (UiType 28)  as a Custom Field to a module?

### DISCUSSION THREAD

**Discussion by LaurensCleyman** [YetiForceCompany/YetiForceCRM#15835] [How to]
Hi all, 

In the Documents module we have a field of UiType 28 - documentsFileUpload:
![image](https://user-images.githubusercontent.com/16014524/160763440-18b9a59e-3da9-45ad-871a-ec4da8dca914.png)

How can we add this type as a custom field to another module, for example the Contacts module?

If I click 'Add Custom Field' then UiType 28 is not available as a selectable field type.

Kind regards,

Laurens

**Comment by rskrzypczak** [comment]
In version 6.4 it will be possible to add an attachment field to any module
![image](https://user-images.githubusercontent.com/10221999/167354007-aa09d5c2-db54-4616-a98a-bea06bbd811c.png)

**Reply by LaurensCleyman** [reply]
You rock!!!

**Comment by s2b-git** [comment]
Not that I am interested in this functionality but in general: what time frame (approximately) do you gents planning release of next version? 
Some people here want to move out of vtiger, so I can keep convincing them showing some light
